doc/architecture/decisions/0019-BLAST-database-files.md

```diff
@@ -0,0 +1,41 @@
+# 19. BLAST database files
+
+Date: 2022-09-11
+
+## Status
+
+Draft
+
+## Context
+
+This document summarises the decisions made when choosing which database options to support for the initial implementation of BLAST on the new Ensembl website.  
+
+## Decision
+
+### Database choices
+BLAST databases are generated from FASTA files copied from the Ensembl and ensemblgenomes FTP sites
+
+#### Genomic sequence
+Unmmasked and softmasked masked, excluding hardmasked. Softmasked sequence has all the sequence, and so the coords are consistent, it’s just that parts are lowercase and will be ignored by BLAST when it’s looking for regions to seed the alignments. An alignment seed can be found near a repeat and the alignment can then be extended into the repeat though. This is based on the idea that you’re probably not looking for things that are wholly contained in repeats. Plus it makes BLAST much faster as it looks at less of the genome.
+Hardmasked sequence has all the repeat regions removed, so it’s essentially like a very extreme version of softmasking that removes anything that looks like a repeat and also changes the coordinates as a result. As a result there will be no alignments to anything classed as a repeat, even through extension of the initial alignment. There are very few scenarios where this would be a good thing and potentially it’s very confusing and misleading.
```

Comment:
Is line 20 supposed to be a separate paragraph? If so, it needs an empty line before it.